### PR TITLE
[ios][file-system] fixed a crash where CFRelease called with a null v…

### PR DESCRIPTION
**Pull request by daxaxelrod**
…alue

# Why

Fixed a bug my team (all credit goes to @ilyausorov) experienced with FileSystem.uploadAsync. [Apple docs](https://developer.apple.com/documentation/corefoundation/1521153-cfrelease) specify that CFRelease can cause a runtime crash if called with NULL, which is what we find to be happening
> Special Considerations
If cf is NULL, this will cause a runtime error and your application will crash.

Reproduced crash here: https://github.com/ilyausorov/test_upload

# How
Adding an if () check before releasing UTI. Fix recommendation taken from [here](https://github.com/rnmods/react-native-document-picker/issues/394#issuecomment-818599489)

# Test Plan
Existing 2 test suites pass. Tested manually and crash no long occurs.

### CODE REVIEW

**Review by barthap — COMMENTED** [review]
Thank you for this fix! 👏 🎉 

Just one suggestion about the changelog and we are ready to merge.

File: packages/expo-file-system/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 ### 💡 Others
 
+## 13.1.1 - 2021-12-09
+
+### 🐛 Bug fixes
+
+- Fixed simulator runtime crash on arm64 devices caused by CFRelease(NULL) ([#15496](https://github.com/expo/expo/pull/15496) by [@daxaxelrod](https://github.com/daxaxelrod))
```
[the remark this answers is not in the document]
Comment (daxaxelrod) [reply packages/expo-file-system/CHANGELOG.md]: Auto changelog creation/versioning is super cool. Change made!

**Review by barthap — APPROVED** [review]
Awesome, thank you!

**Review by bbarthec — APPROVED** [review]
Hello @daxaxelrod 👋 
Thank you for this contribution 🥳 
The code looks good to me 💪 
Merging it now 😁

**Pull request merged**